### PR TITLE
os-2374

### PR DESCRIPTION
**Pull request by dschwabeS11**
- Less steps to working openstack client installation
- Infobox about minimum client version to be able to work with multiple regions

### CODE REVIEW

**Comment by stefreak** [comment]
👍

File: german/docs/tutorials/openstack-cli.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ Dieses Tutorial dient als Anleitung für die Installation des Openstack CLI (Com
 *In dieser Anleitung gehen wir davon aus, dass bisher nichts von den benötigten Programmen installiert wurde.
 Für den Fall das bereits Bestandteile installiert sind, überspringe den jeweiligen Punkt.*
 
+<div class="alert alert-dismissible alert-info">
+    <h4 class="alert-heading">Benötigte OpenStack Client Version die mit der SysEleven Cloud kompatibel ist</h4>
+    OpenStack Client Version 3.13.x ist die minimale Version um mit mehreren Regionen arbeiten zu können<br>
```
Comment (bpetermannS11) [inline german/docs/tutorials/openstack-cli.md]: Punkt am Ende

File: german/docs/tutorials/openstack-cli.md [hunk 2]
```diff
@@ -20,6 +20,12 @@ Dieses Tutorial dient als Anleitung für die Installation des Openstack CLI (Com
...
+    Bitte sicherstellen dass die neuste stabile Version installiert ist.
```
Comment (bpetermannS11) [inline german/docs/tutorials/openstack-cli.md]: neueste

**Comment by stefreak** [comment]
@dschwabeS11 can you fix the stuff that @bpetermannS11 found in another PR? sorry for merging it like that ;)

**Comment by dschwabeS11** [comment]
Was fixed with https://github.com/syseleven/syselevenstack-documentation/pull/68

**Pull request merged**